packages/CanvasIFrameModule/custom/themes/custom.less

```diff
@@ -0,0 +1,3 @@
+.full-width {
+  width: 100%;
+}
```

Comment:
nitpick: add a newline here to remove warning. Git doesn't like it when you don't end a file with a new line character.